ipa-core/src/report/hybrid.rs

```diff
@@ -50,8 +46,37 @@ pub enum InvalidHybridReportError {
     DeserializationError(&'static str, #[source] BoxError),
     #[error("report is too short: {0}, expected length at least: {1}")]
     Length(usize, usize),
+    #[error("unknown event type: {0}. Only 0 and 1 are allowed")]
+    UnknownEventType(u8),
+    #[error("Incorrect hybrid info type: Expected {0}")]
+    WrongInfoType(&'static str),
 }
 
+/*#[derive(thiserror::Error, Debug)]
```

Comment:
there are a few commented out lines like this scattered though out that should be cleaned up